.github/workflows/steam.yml

```diff
@@ -36,6 +36,12 @@ jobs:
           restore-keys: |
             ccache-win64-cross-msvc-develop-${{ github.event.inputs.commit_hash }}
             ccache-win64-cross-msvc
+      - name: Restore steam SDK
+        uses: actions/cache@v3
+        with:
+          path: depends/steam/steamworks_sdk_156.zip
```

Comment:
shouldn't this be a *directory* rather than a file name? i.e. `depends/steam`?

Reply:
per github docs either is fine, see no reason not to cache just the file

![image](https://user-images.githubusercontent.com/1445859/234422455-6ff68db3-d40c-4fe2-81ed-b4fd6b7d33e3.png)

Reply:
fair enough. let's give this a try